.github/workflows/callable_test-automation.yaml

```diff
@@ -372,17 +363,17 @@ jobs:
         run: |
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
-
   get-test-modules:
     name: Get Test Modules
     runs-on: ubuntu-latest
     outputs:
       modules: ${{ steps.get-module-list.outputs.file_names }}
+      payment-modules: ${{ steps.get-payment-module-list.outputs.file_names }}
     steps:
-      - name: Git checkout hyperledger/indy-test-automation
+      - name: Git checkout VladimirWork/indy-test-automation
```

Comment:
Also noticed that this PR is still pointing to the GHA on your fork. 
While in development and finalizing the PR this is fine, but should be taken care of before the merge.

Reply:
This is for testing purpose only, I will roll back this right before merging.